service/src/main/java/bio/terra/pipelines/db/entities/Pipeline.java

```diff
@@ -116,7 +121,8 @@ public String toString() {
         .add("workspaceId=" + workspaceId)
         .add("workspaceProject=" + workspaceProject)
         .add("workspaceName=" + workspaceName)
-        .add("workspaceStorageContainerUrl=" + workspaceStorageContainerUrl)
+        .add("workspaceStorageContainerUrl=" + workspaceStorageContainerName)
```

Comment:
`workspaceStorageContainerUrl` -> `workspaceStorageContainerName` in the string part

Reply:
nice catch, thanks!